src/com/synopsys/integration/pipeline/utilities/MavenUtils.groovy

```diff
@@ -60,6 +60,11 @@ public class MavenUtils implements ToolUtils, Serializable {
         return version
     }
 
+    @Override
+    public String updateVersionForRelease(boolean runRelease, boolean runQARelease, String versionUpdateCommand) {
+        return updateVersionForRelease(runRelease, runQARelease)
+    }
+
```

Comment:
Can you just add a comment in here to explain why this isn't using versionUpdateCommand or doing anything different than the existing `updateVersionForRelease(boolean runRelease, boolean runQARelease)`